app/views/events/_form.html.haml

```diff
@@ -1,7 +1,9 @@
 = simple_form_for resource do |f|
   .section
-    = render 'form_base', f: f
```

Comment:
Okay, I'll take a look. There are simpler ways to achieve that. The way you did it introduces quite some code duplication :sunglasses: